go-apps/meep-rnis/server/rnis.go

```diff
@@ -44,6 +44,8 @@ const logModuleRNIS string = "meep-rnis"
 //const module string = "rnis"
 var redisAddr string = "meep-redis-master.default.svc.cluster.local:6379"
 var influxAddr string = "http://meep-influxdb.default.svc.cluster.local:8086"
+var postgisHost string = "meep-postgis.default.svc.cluster.local"
+var postgisPort string = "5432"
 
```

Comment:
Should a high level service such as RNIS connect directly to the geospatial database?
Was expecting that it would use the gis package that hides details about the database.